server/datastore/mysql/software.go

```diff
@@ -402,6 +402,7 @@ func listSoftwareDB(
 				cve.CVSSScore = &result.CVSSScore
 				cve.EPSSProbability = &result.EPSSProbability
 				cve.CISAKnownExploit = &result.CISAKnownExploit
+				cve.CVEPublished = &result.CVEPublished
```

Comment:
Not for now, but eventually we should rename `opts.IncludeCVEScores` to something like `opts.IncludePremiumMetadata` (the published date is not a score per-se)

Reply:
+1